connectors/src/connectors/intercom/index.ts

```diff
@@ -252,6 +250,23 @@ export async function retrieveIntercomConnectorPermissions({
   }
 }
 
+async function startWorkflowIfNecessary(
```

Comment:
I'm not a masssssive fan of having these kind of helpers if they are only used in one place. I know who this is coming from @flvndvd :)

IMHO it just makes the code harder to understand but I can def live with it. Just sharing my feeling :)